go-controller/pkg/ovn/default_network_controller.go

```diff
@@ -160,7 +160,10 @@ func newDefaultNetworkControllerCommon(cnci *CommonNetworkControllerInfo,
 	if addressSetFactory == nil {
 		addressSetFactory = addressset.NewOvnAddressSetFactory(cnci.nbClient)
 	}
-	svcController, svcFactory := newServiceController(cnci.client, cnci.nbClient, cnci.recorder)
+	svcController, svcFactory, err := newServiceController(cnci.client, cnci.nbClient, cnci.recorder)
+	if err != nil {
+		klog.Errorf("Unable to create new service controller: %v", err)
```

Comment:
Should we return error here? If not, make it fatal

Reply:
not resolved, this needs to return error